e2e-tests/e2e-test-runner.go

```diff
@@ -10,10 +10,15 @@ import (
 	"testing"
 
 	"code-intelligence.com/cifuzz/integration-tests/shared"
+	"code-intelligence.com/cifuzz/internal/cmdutils/login"
+	"code-intelligence.com/cifuzz/internal/tokenstorage"
 	"code-intelligence.com/cifuzz/pkg/cicheck"
 	"code-intelligence.com/cifuzz/util/fileutil"
 )
 
+const ciServerToUseForE2ETests = "https://app.code-intelligence.com/"
```

Comment:
I vote for using the staging server
```suggestion
const CIServerToUseForE2ETests = "https://app.staging.code-intelligence.com/"
```

Reply:
honestly I vote for not using any remote server at all... but if we use one, we should use `staging` and talk to the web team about it

Reply:
> honestly I vote for not using any remote server at all

I'm not totally opposed to it, yet do we check that "full-circle" cifuzz<->API interaction anywhere else currently?

---

> I vote for using the staging server

> but if we use one, we should use staging and talk to the web team about it

Why would you use a staging?

Reply:
> I'm not totally opposed to it, yet do we check that "full-circle" cifuzz<->API interaction anywhere else currently?

I am not sure about this... maybe in the core repo. @simonresch might know
But I agree we need some real e2e/integration tests, but I would prefer them setup independently from this repo, as this is only one part of the whole product. And even if we test against a real (prod) server it is not really covering the server part as there is no actual reporting / logging /... with the server team involved
Also it would make our pipeline useless if there is a downtime

> Why would you use a staging?

To make sure we are not causing any trouble in production, like distorted usage data or DDoSing ourself... should not happen, but you never know. Also having test data (accounts, tokens, projects...) in our prod setup just feels wrong

I am aware that there are reasons for using prod (less downtime, matches what the user is actually using). But I am not sure about if it is worth doing this

Reply:
> I am not sure about this... maybe in the core repo.

We currently have tests in the core repository that create a bundle with cifuzz and then start bundle execution via `cictl`. The test asserts certain findings and coverage information. https://gitlab.code-intelligence.com/code-intelligence/core/-/blob/master/backend/scripts/smoke_test/core.clj#L728
I'm happy to add more test cases there.

The limitation of the test in the core repository currently is that you would not be able to run it easily for every commit or PR, since we use a fixed `cifuzz` release which is manually updated.

Reply:
Switched to staging